lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -56,6 +56,7 @@ def indexables
         load_path_entry = T.let(nil, T.nilable(String))
 
         Dir.glob(pattern, File::FNM_PATHNAME | File::FNM_EXTGLOB).map! do |path|
+          path = File.expand_path(path)
```

Comment:
Expand path doesn't do anything if the path is already absolute, right? Trying to confirm that we'll continue supporting glob patterns based on absolute paths (e.g.: `/Users/username/somewhere/**/*.rb`)

Reply:
No it doesn't do anything in that case:

```
irb(main):005> File.expand_path("lib")
=> "/Users/hung-wulo/src/github.com/ruby/irb/lib"
irb(main):006> File.join(Dir.pwd, "lib")
=> "/Users/hung-wulo/src/github.com/ruby/irb/lib"
irb(main):007> File.expand_path(File.join(Dir.pwd, "lib"))
=> "/Users/hung-wulo/src/github.com/ruby/irb/lib"
```